### PR TITLE
Perfecto integration documentation in full

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/integrations/perfecto-integration.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+---
+title: Perfecto integration
+description: How to integrate your workflows with Perfecto using codemagic.yaml
+weight: 13
+---
+
+**Perfecto** is a web-based Software as a Service (SaaS) platform that allows mobile application developers and QA Engineers to work with services such as advanced automation, monitoring and testing services. It is possible to integrate with Perfecto directly from your **codemagic.yaml**
```
Comment (nevercode-kevin) [inline content/integrations/perfecto-integration.md:7]:
```suggestion
**Perfecto** is a web-based platform that allows mobile application developers and QA Engineers to work with services such as advanced automation, monitoring, and testing services. It is possible to integrate with Perfecto directly from your **codemagic.yaml**
```

File: content/integrations/perfecto-integration.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+Signing up with [Perfecto](https://www.perfecto.io/) is required in order to get credentials that are needed during an upload process. 
+
+Usin the following cURL script in a pre-build script(a script that is run after executing build commands in yaml), **.apk**, **.aab** and **.ipa** binaries can be uploaded to the Perfecto platform:
```
Comment (nevercode-kevin) [inline content/integrations/perfecto-integration.md:11]:
```suggestion
Usin gthe following cURL script in a post-build script(a script that is run after executing build commands in yaml), **.apk**, **.aab** and **.ipa** binaries can be uploaded to the Perfecto platform:
```

File: content/integrations/perfecto-integration.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+```
+curl "https://web.app.perfectomobile.com/repository/api/v1/artifacts" -H "Perfecto-Authorization: $PERFECTO_TOKEN" -H "Content-Type: multipart/form-data" -F "requestPart={\"artifactLocator\":\"PRIVATE:app.aab\",\"artifactType\":\"ANDROID\",\"override\":true}" -F "inputStream=@/path/to/your_binary"
+```
+
+**PERFECTO_TOKEN** can found in the Perfecto UI with your account. Environment variables can be added in the Codemagic web app using the ‘Environment variables’ tab. You can then and import your variable groups into your codemagic.yaml. For example, if you named your variable group ‘browserstack_credentials’, you would import it as follows:
```
Comment (nevercode-kevin) [inline content/integrations/perfecto-integration.md:17]:
```suggestion
**PERFECTO_TOKEN** can be found in the Perfecto UI with your account. Environment variables can be added in the Codemagic web app using the ‘Environment variables tab. You can then import your variable groups into your codemagic.yaml. For example, if you named your variable group ‘perfecto_credentials’, you would import it as follows:
```

File: content/integrations/perfecto-integration.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+	"runUITests":true,
+	"runUnitTests":false,
+	"installationDetails": {
+		"resign": true
+	  },
+	  
+  	"numOfDevices": 2
+  }
+```
+4. In order to successfully upload files and enable test automation, the following commands needs to be executed:
```
Comment (nevercode-kevin) [inline content/integrations/perfecto-integration.md:200]:
```suggestion
4. In order to successfully upload files and enable test automation, the following commands need to be executed:
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Comment by icarusdust** [comment]
Done!

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**